docs/configuration/admin-interface.md

```diff
@@ -0,0 +1,54 @@
+# Admin interface
+
+!!! tldr "Django docs"
+
+    [The Django admin site](https://docs.djangoproject.com/en/5.0/ref/contrib/admin/)
+
+The configuration values that can be stored in the application database are managed directly in the [Django Admin interface](./admin-interface.md) at the `/admin` endpoint.
+
+Single sign-on authentication is supported by the admin interface using [`django-google-sso`](https://megalus.github.io/django-google-sso/).
+
+
+## Environment variables
+
+!!! tldr "`django-google-sso` docs"
+
+    [All settings for `django-google-sso`](https://megalus.github.io/django-google-sso/settings/)
+
+!!! info "`settings.py`"
+
+    Configuration of `django-google-sso` in [Benefits settings](https://github.com/cal-itp/benefits/blob/4625f250f48eb9071d9b8fdc79e4dfc6cb2ee625/benefits/settings.py#L61-L74)
```

Comment:
Should link to the version in the `main` branch.

Reply:
An issue I've seen with using a version in a branch is that the line numbers will become out of date whenever the file is inevitably updated in that branch, so that was my reasoning for using a specific commit. But if we'd prefer to use `main` regardless, I can change it.

Reply:
That's a good point. I was thinking it's better to link to the wrong line in the current file vs. the correct line in a potentially out of date version of the file?

Reply:
Yeah that makes sense. I updated the link in https://github.com/cal-itp/benefits/pull/2220/commits/f9c32a8bbc3ce38255902e7866390960e9d15fe0